LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDevice.cs

```diff
@@ -205,6 +233,14 @@ public async Task<bool> InitializeAsync()
                     this.Deduplication = mode;
                 }
 
+                if (twin.Properties.Desired.Contains(TwinProperty.ClassType))
```

Comment:
I understand there are no other class types we use explicitly at the moment, but woudln't it make sense to add the parsing for all values in the enum?

Reply:
Current behavior will treat any value other than "C" as a class A device. Notice that there is a 'B' defined in the enum, but it is not yet supported. I have chosen to assume "A" if any invalid value is set.

However, if the requirement is to fail if the device class is set to "B" I will have to make changes.

@ronniesa what is your take? Fail or switch silently to class A device if the value is B or invalid?